pybaseball/top_prospects.py

```diff
@@ -3,11 +3,18 @@
 
 from . import cache
 
+from pybaseball import teamid_lookup
+
 
 @cache.df_cache()
 def top_prospects(teamName=None, playerType=None):
-    teamUrl = "" if teamName == None else teamName.lower() + '/'
```

Comment:
Ah, I see the sanitizing here, which is also nice, but would be cleaner to rope into the function where the teamID is actually gathered so if you want to just import that, you can.